app/allocation/steps/person-assign.js

```diff
@@ -0,0 +1,106 @@
+const { create } = require('../../move/steps')
+const controllers = require('../controllers/person-assign')
+
+const personAssign = { ...create }
+
+const updateAssignStep = (key, props) => {
+  personAssign[key] = { ...personAssign[key], ...props }
+}
+
+updateAssignStep('/person-lookup-prison-number', {
+  controller: controllers.PersonSearch,
+  next: 'person-lookup-results',
+})
+
+updateAssignStep('/person-lookup-results', {
+  controller: controllers.PersonSearchResults,
+  next: 'agreement-status',
+})
+
+updateAssignStep('/agreement-status', {
+  next: 'release-status',
+})
+
+updateAssignStep('/release-status', {
+  controller: controllers.Assessment,
+  next: 'special-vehicle',
+})
+
+updateAssignStep('/special-vehicle', {
+  controller: controllers.Assessment,
+  pageTitle: 'moves::steps.health_information.heading',
+  fields: ['special_vehicle_check'],
+  next: [
+    {
+      field: 'special_vehicle_check',
+      value: 'true',
+      next: 'no-special-vehicle',
+    },
+    'save',
+  ],
+})
+
+updateAssignStep('/no-special-vehicle', {
+  templatePath: 'allocation/views/person-assign/',
+  template: 'no-special-vehicle',
+  pageTitle: 'moves::steps.person_search.heading',
+})
+
+updateAssignStep('/save', {
+  controller: controllers.Save,
+  next: 'confirmation',
+})
+
+updateAssignStep('/confirmation', {
```

Comment:
Would it be simpler to redirect back the allocation with a success flash message for this journey do you think?

Reply:
That's how it's been designed.

Reply:
Okay, maybe actually just redirect to the confirmation page of the move that already exists. 

And tweak the language if the move is related to an allocation. Saves having to duplicate that template, view, etc.